### DISCUSSION TITLE
Delta Green (Lesser) by @lithedreamer

### DISCUSSION THREAD

**Discussion by sigrunixia** [Obsidian-TTRPG-Community/fantasy-statblocks#167] [Show and tell]
This thread is for @lithedreamer and their Delta Green (Lesser) Statblock.

---

### Delta Green

This should work for _unnatural entities_ your players might encounter. There aren't a lot of pretty options for rolling d100s for these stats that I can think of, so I opted to mark as much of the layout for markdown instead.

![Screen Shot 2022-11-17 at 03 25 37](https://user-images.githubusercontent.com/2753891/202434150-ea67c627-2422-42a8-bbd9-a724f898e497.png)

[DG-Layout.zip](https://github.com/valentine195/obsidian-5e-statblocks/files/10030911/DG-Layout.zip)

---

**Original Thread**: https://github.com/valentine195/fantasy-statblocks/discussions/41#discussioncomment-4166497

**Comment by anoriginalmoniker** [comment]
This is great! Do you have a bunch of these statblocks typed out already? I'd love to copy from you to save some work. 😜
